carbonmark-api/src/utils/helpers/users.utils.ts

```diff
@@ -58,22 +65,53 @@ export const getUserDocumentsByIds = async (
   }
 };
 
+/**
+ * Provide a sorted deduplicated array of lower-case wallet addresses
+ * Get a Map of user profiles. Performance-optimized firestore query
+ */
+export const getUserProfilesByIds = async (params: {
```

Comment:
This is looking real similar to the `getUserDocumentsByIds` util. Can we use just one?